specification/network/resource-manager/Microsoft.Network/preview/2021-02-01-preview/networkManagerScopeConnections.json

```diff
@@ -0,0 +1,378 @@
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/networkManagers/{networkManagerName}/listPendingNetworkManagerConnections": {
+      "post": {
+        "tags": [
+          "ScopeConnections"
+        ],
+        "operationId": "PendingNetworkManagerConnection_List",
+        "description": "List all pending network manager connections sent from a subscription or management group.",
+        "parameters": [
+          {
+            "$ref": "./network.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "./networkManager.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "./networkManager.json#/parameters/NetworkManagerNameParameter"
+          },
+          {
+            "$ref": "./network.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK - Returns a list of the network manager connections sent from a subscription or management group.",
+            "schema": {
+              "$ref": "./networkManagerConnections.json#/definitions/PendingNetworkManagerConnectionListResult"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "./network.json#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "List Pending Network Manager Connections Sent From A Subscription or Management Group": {
+            "$ref": "./examples/NetworkManagerPendingConnectionList.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "ScopeConnection": {
+      "type": "object",
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/ScopeConnectionProperties",
+          "description": "The scope connection properties"
+        },
+        "etag": {
+          "readOnly": true,
+          "type": "string",
+          "description": "A unique read-only string that changes whenever the resource is updated."
+        },
+        "systemData": {
+          "readOnly": true,
+          "description": "The system metadata related to this resource.",
+          "$ref": "./network.json#/definitions/SystemData"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "./network.json#/definitions/ProxyResource"
+        }
+      ],
+      "description": "The Scope Connections resource"
+    },
+    "ScopeConnectionProperties": {
+      "type": "object",
+      "properties": {
+        "tenantId": {
+          "type": "string",
+          "description": "Tenant ID."
+        },
+        "managementGroups": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ScopeConnectionManagementGroup"
```

Comment:
I don't think these need to be two different objects. Mg and Sub will have the exact same schema. And, the id property is a string anyway. Consolidate into a single object